force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,9 +29,12 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+            if (x != null && y != null && x > y) {
```

Comment:
```suggestion
        if (x != null && y != null && x > y) {
```

Reply:
Be careful of your indenting. The if statement should be lined up with the bracket that closes it.